src/shared/ui/composables/useErrorHandlers.ts

```diff
@@ -18,5 +19,11 @@ export const useErrorHandlers = () => {
     notifications.error(errorMessage);
   }
 
-  return { handleUnknownError };
+  function handleZodError(error: ZodError) {
+    console.error(error);
+
+    notifications.error(error.errors[0].code);
+  }
```

Comment:
It can be a single universal error handler that dynamically checks whether `error: unknown` is e.g. an `instanceof ZodError`.

Reply:
Also, there is `.format` method on `ZodError`, please check the docs.

Reply:
I use `.parse` method instead of `.safeParse`. The following gives me an empty array:
```
error.format()._errors
```
I would prefer keep using `.parse` method so lets keep it as is. With that, we receive quite clear message about an error.